site/profile/manifests/app/splunk/README.md

```diff
@@ -0,0 +1,2 @@
+# Sets up Splunk Server - started 6/15/2017 - TLS
+# Specify settings in the module to start with, with plans to move it to hiera
```

Comment:
Don't know if this README makes sense to include. Maybe rewrite it to be more useful?